src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menArray = people.filter(person => person.sex === 'm');
+
+  const menRangeArray = !century ? menArray
+    : menArray.filter(man => Math.ceil(man.died / 100) === century);
```

Comment:
```suggestion
  const menRangeArray = century
  	? menArray.filter(man => Math.ceil(man.died / 100) === century)
    : menArray;
```

Reply:
Formatting is very important
mate-academy.github.io/style-guides/javascript.html#comparison--nested-ternaries